bin/start

```diff
@@ -1,22 +1,36 @@
 #!/usr/bin/env bash
 
+if [ -f .env ]; then
+  export $(echo $(cat .env | sed 's/#.*//g'| xargs) | envsubst)
+fi
+
 PORT="${PORT:=3000}"
-PGHOST=postgres
-PGPORT=5432
-RSHOST=redis
-RSPORT=6379
 
-echo "Greenlight-v3 starting on port: $PORT"
+# Parse Rails DATABASE and REDIS urls to get host and port
+pgProtocol="$(echo $DATABASE_URL | grep :// | sed -e's,^\(.*://\).*,\1,g')" # extract the protocol
+pgUrl="$(echo ${DATABASE_URL/$pgProtocol/})" # remove the protocol
```

Comment:
I think using echo here isn't required since this is a bash operation,
`pgUrl=${DATABASE_URL/$pgProtocol/}`